src/shared/types.ts

```diff
@@ -75,6 +75,7 @@ export interface BitcoinNode extends CommonNode {
 }
 
 export interface TapNode extends CommonNode {
+  lndName: string;
```

Comment:
This shouldn't be here `TapNode` is a generic type which `TapdNode` extends and has the `lndName` field on it. You should remove this and use that type instead.

The original thought here was that `tapd` may be one of many Taproot Asset nodes that Polar supports in the future. The `lndName` field is specific to the `tapd` implementation but may not be required for others. Think of it like how `LndNode` and `EclairNode` both extend `LightningNode`. So there may come a day where there's `TapdNode` and `SomeFutureTapNode` that both extend `TapNode`.